R/filter.R

```diff
@@ -17,11 +17,15 @@
 #'  the value of the object in the expression (to be portable
 #'  between sessions). See the examples.
 #'
-#'  # Tidying
+#' # Tidying
 #'
-#'  When you [`tidy()`][tidy.recipe()] this step, a tibble with column
-#'  `terms` which contains the conditional statements is returned.
-#'  These expressions are text representations and are not parsable.
+#' When you [`tidy()`][tidy.recipe()] this step, a tibble is returned with 2
+#' columns `terms` and `id`:
+#'
+#' \describe{
+#'   \item{terms}{character, the selectors or variables selected}
```

Comment:
This used to be documented as:

> #'  `terms` which contains the conditional statements is returned.
> #'  These expressions are text representations and are not parsable.

Is the new description still accurate?

Reply:
fixed in [`3b61af0` (#1230)](https://github.com/tidymodels/recipes/pull/1230/commits/3b61af0dc61b2122e402a8c334dd85451565b942)